runelite-client/src/main/java/net/runelite/client/plugins/devtools/LocationOverlay.java

```diff
@@ -64,6 +64,12 @@ public Dimension render(Graphics2D graphics)
 
 		if (client.isInInstancedRegion())
 		{
+			WorldPoint instancePoint = WorldPoint.fromLocalInstance(client, localPoint);
```

Comment:
`fromLocalInstance` can never be null, maybe we can mark it as nonnull

Reply:
removed null check